<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QScrollArea" name="StatusVariables">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>193</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>435</width>
         <height>191</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="PhyReadyCnt">
          <property name="text">
           <string> PhyReadyCnt </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="caLineEdit" name="$(DEVICE):PhyReadyCnt">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):PhyReadyCnt</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="RxPauseCnt">
          <property name="text">
           <string> RxPauseCnt </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="caLineEdit" name="$(DEVICE):RxPauseCnt">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):RxPauseCnt</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="TxPauseCnt">
          <property name="text">
           <string> TxPauseCnt </string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="caLineEdit" name="$(DEVICE):TxPauseCnt">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):TxPauseCnt</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="RxCountEnCnt">
          <property name="text">
           <string> RxCountEnCnt </string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="caLineEdit" name="$(DEVICE):RxCountEnCnt">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):RxCountEnCnt</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="RxOverFlowCnt">
          <property name="text">
           <string> RxOverFlowCnt </string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="caLineEdit" name="$(DEVICE):RxOverFlowCnt">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):RxOverFlowCnt</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="RxCrcErrorCnt">
          <property name="text">
           <string> RxCrcErrorCnt </string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="caLineEdit" name="$(DEVICE):RxCrcErrorCnt">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):RxCrcErrorCnt</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="TxCountEnCnt">
          <property name="text">
           <string> TxCountEnCnt </string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="caLineEdit" name="$(DEVICE):TxCountEnCnt">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):TxCountEnCnt</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="TxUnderRunCnt">
          <property name="text">
           <string> TxUnderRunCnt </string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="caLineEdit" name="$(DEVICE):TxUnderRunCnt">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):TxUnderRunCnt</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="TxNotReadyCnt">
          <property name="text">
           <string> TxNotReadyCnt </string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="caLineEdit" name="$(DEVICE):TxNotReadyCnt">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):TxNotReadyCnt</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="TxDisable">
          <property name="text">
           <string> TxDisable </string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="caLineEdit" name="$(DEVICE):TxDisable">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):TxDisable</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="SigDet">
          <property name="text">
           <string> SigDet </string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="caLineEdit" name="$(DEVICE):SigDet">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):SigDet</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="TxFault">
          <property name="text">
           <string> TxFault </string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="caLineEdit" name="$(DEVICE):TxFault">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):TxFault</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="GtTxRst">
          <property name="text">
           <string> GtTxRst </string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="caLineEdit" name="$(DEVICE):GtTxRst">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):GtTxRst</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="GtRxRst">
          <property name="text">
           <string> GtRxRst </string>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="caLineEdit" name="$(DEVICE):GtRxRst">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):GtRxRst</string>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QLabel" name="RstCntDone">
          <property name="text">
           <string> RstCntDone </string>
          </property>
         </widget>
        </item>
        <item row="14" column="1">
         <widget class="caLineEdit" name="$(DEVICE):RstCntDone">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):RstCntDone</string>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QLabel" name="Qplllock">
          <property name="text">
           <string> Qplllock </string>
          </property>
         </widget>
        </item>
        <item row="15" column="1">
         <widget class="caLineEdit" name="$(DEVICE):Qplllock">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):Qplllock</string>
          </property>
         </widget>
        </item>
        <item row="16" column="0">
         <widget class="QLabel" name="TxRstdone">
          <property name="text">
           <string> TxRstdone </string>
          </property>
         </widget>
        </item>
        <item row="16" column="1">
         <widget class="caLineEdit" name="$(DEVICE):TxRstdone">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):TxRstdone</string>
          </property>
         </widget>
        </item>
        <item row="17" column="0">
         <widget class="QLabel" name="RxRstdone">
          <property name="text">
           <string> RxRstdone </string>
          </property>
         </widget>
        </item>
        <item row="17" column="1">
         <widget class="caLineEdit" name="$(DEVICE):RxRstdone">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):RxRstdone</string>
          </property>
         </widget>
        </item>
        <item row="18" column="0">
         <widget class="QLabel" name="TxUsrRdy">
          <property name="text">
           <string> TxUsrRdy </string>
          </property>
         </widget>
        </item>
        <item row="18" column="1">
         <widget class="caLineEdit" name="$(DEVICE):TxUsrRdy">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):TxUsrRdy</string>
          </property>
         </widget>
        </item>
        <item row="19" column="0">
         <widget class="QLabel" name="Status">
          <property name="text">
           <string> Status </string>
          </property>
         </widget>
        </item>
        <item row="19" column="1">
         <widget class="caLineEdit" name="$(DEVICE):Status">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):Status</string>
          </property>
         </widget>
        </item>
        <item row="20" column="0">
         <widget class="QLabel" name="FLAG_PhyReady">
          <property name="text">
           <string> FLAG_PhyReady </string>
          </property>
         </widget>
        </item>
        <item row="20" column="1">
         <widget class="caLineEdit" name="$(DEVICE):FLAG_PhyReady">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):FLAG_PhyReady</string>
          </property>
         </widget>
        </item>
        <item row="21" column="0">
         <widget class="QLabel" name="FLAG_RxPause">
          <property name="text">
           <string> FLAG_RxPause </string>
          </property>
         </widget>
        </item>
        <item row="21" column="1">
         <widget class="caLineEdit" name="$(DEVICE):FLAG_RxPause">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):FLAG_RxPause</string>
          </property>
         </widget>
        </item>
        <item row="22" column="0">
         <widget class="QLabel" name="FLAG_TxPause">
          <property name="text">
           <string> FLAG_TxPause </string>
          </property>
         </widget>
        </item>
        <item row="22" column="1">
         <widget class="caLineEdit" name="$(DEVICE):FLAG_TxPause">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):FLAG_TxPause</string>
          </property>
         </widget>
        </item>
        <item row="23" column="0">
         <widget class="QLabel" name="FLAG_RxCountEn">
          <property name="text">
           <string> FLAG_RxCountEn </string>
          </property>
         </widget>
        </item>
        <item row="23" column="1">
         <widget class="caLineEdit" name="$(DEVICE):FLAG_RxCountEn">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):FLAG_RxCountEn</string>
          </property>
         </widget>
        </item>
        <item row="24" column="0">
         <widget class="QLabel" name="FLAG_RxOverFlow">
          <property name="text">
           <string> FLAG_RxOverFlow </string>
          </property>
         </widget>
        </item>
        <item row="24" column="1">
         <widget class="caLineEdit" name="$(DEVICE):FLAG_RxOverFlow">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):FLAG_RxOverFlow</string>
          </property>
         </widget>
        </item>
        <item row="25" column="0">
         <widget class="QLabel" name="FLAG_RxCrcError">
          <property name="text">
           <string> FLAG_RxCrcError </string>
          </property>
         </widget>
        </item>
        <item row="25" column="1">
         <widget class="caLineEdit" name="$(DEVICE):FLAG_RxCrcError">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):FLAG_RxCrcError</string>
          </property>
         </widget>
        </item>
        <item row="26" column="0">
         <widget class="QLabel" name="FLAG_TxCountEn">
          <property name="text">
           <string> FLAG_TxCountEn </string>
          </property>
         </widget>
        </item>
        <item row="26" column="1">
         <widget class="caLineEdit" name="$(DEVICE):FLAG_TxCountEn">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):FLAG_TxCountEn</string>
          </property>
         </widget>
        </item>
        <item row="27" column="0">
         <widget class="QLabel" name="FLAG_TxUnderRun">
          <property name="text">
           <string> FLAG_TxUnderRun </string>
          </property>
         </widget>
        </item>
        <item row="27" column="1">
         <widget class="caLineEdit" name="$(DEVICE):FLAG_TxUnderRun">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):FLAG_TxUnderRun</string>
          </property>
         </widget>
        </item>
        <item row="28" column="0">
         <widget class="QLabel" name="FLAG_TxNotReady">
          <property name="text">
           <string> FLAG_TxNotReady </string>
          </property>
         </widget>
        </item>
        <item row="28" column="1">
         <widget class="caLineEdit" name="$(DEVICE):FLAG_TxNotReady">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):FLAG_TxNotReady</string>
          </property>
         </widget>
        </item>
        <item row="29" column="0">
         <widget class="QLabel" name="FLAG_TxDisable">
          <property name="text">
           <string> FLAG_TxDisable </string>
          </property>
         </widget>
        </item>
        <item row="29" column="1">
         <widget class="caLineEdit" name="$(DEVICE):FLAG_TxDisable">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):FLAG_TxDisable</string>
          </property>
         </widget>
        </item>
        <item row="30" column="0">
         <widget class="QLabel" name="FLAG_SigDet">
          <property name="text">
           <string> FLAG_SigDet </string>
          </property>
         </widget>
        </item>
        <item row="30" column="1">
         <widget class="caLineEdit" name="$(DEVICE):FLAG_SigDet">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):FLAG_SigDet</string>
          </property>
         </widget>
        </item>
        <item row="31" column="0">
         <widget class="QLabel" name="FLAG_TxFault">
          <property name="text">
           <string> FLAG_TxFault </string>
          </property>
         </widget>
        </item>
        <item row="31" column="1">
         <widget class="caLineEdit" name="$(DEVICE):FLAG_TxFault">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):FLAG_TxFault</string>
          </property>
         </widget>
        </item>
        <item row="32" column="0">
         <widget class="QLabel" name="FLAG_GtTxRst">
          <property name="text">
           <string> FLAG_GtTxRst </string>
          </property>
         </widget>
        </item>
        <item row="32" column="1">
         <widget class="caLineEdit" name="$(DEVICE):FLAG_GtTxRst">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):FLAG_GtTxRst</string>
          </property>
         </widget>
        </item>
        <item row="33" column="0">
         <widget class="QLabel" name="FLAG_GtRxRst">
          <property name="text">
           <string> FLAG_GtRxRst </string>
          </property>
         </widget>
        </item>
        <item row="33" column="1">
         <widget class="caLineEdit" name="$(DEVICE):FLAG_GtRxRst">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):FLAG_GtRxRst</string>
          </property>
         </widget>
        </item>
        <item row="34" column="0">
         <widget class="QLabel" name="FLAG_RstCntDone">
          <property name="text">
           <string> FLAG_RstCntDone </string>
          </property>
         </widget>
        </item>
        <item row="34" column="1">
         <widget class="caLineEdit" name="$(DEVICE):FLAG_RstCntDone">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):FLAG_RstCntDone</string>
          </property>
         </widget>
        </item>
        <item row="35" column="0">
         <widget class="QLabel" name="FLAG_Qplllock">
          <property name="text">
           <string> FLAG_Qplllock </string>
          </property>
         </widget>
        </item>
        <item row="35" column="1">
         <widget class="caLineEdit" name="$(DEVICE):FLAG_Qplllock">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):FLAG_Qplllock</string>
          </property>
         </widget>
        </item>
        <item row="36" column="0">
         <widget class="QLabel" name="FLAG_TxRstdone">
          <property name="text">
           <string> FLAG_TxRstdone </string>
          </property>
         </widget>
        </item>
        <item row="36" column="1">
         <widget class="caLineEdit" name="$(DEVICE):FLAG_TxRstdone">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):FLAG_TxRstdone</string>
          </property>
         </widget>
        </item>
        <item row="37" column="0">
         <widget class="QLabel" name="FLAG_RxRstdone">
          <property name="text">
           <string> FLAG_RxRstdone </string>
          </property>
         </widget>
        </item>
        <item row="37" column="1">
         <widget class="caLineEdit" name="$(DEVICE):FLAG_RxRstdone">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):FLAG_RxRstdone</string>
          </property>
         </widget>
        </item>
        <item row="38" column="0">
         <widget class="QLabel" name="FLAG_TxUsrRdy">
          <property name="text">
           <string> FLAG_TxUsrRdy </string>
          </property>
         </widget>
        </item>
        <item row="38" column="1">
         <widget class="caLineEdit" name="$(DEVICE):FLAG_TxUsrRdy">
          <property name="channel" stdset="0">
           <string notr="true">$(DEVICE):FLAG_TxUsrRdy</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QScrollArea" name="ConfigurationVariables">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>435</width>
         <height>190</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>455</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>caTextEntry</class>
   <extends>caLineEdit</extends>
   <header>caTextEntry</header>
  </customwidget>
  <customwidget>
   <class>caLineEdit</class>
   <extends>QLineEdit</extends>
   <header>caLineEdit</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>

